python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -366,45 +367,53 @@ def _build_column_lineage_metadata(
             "run", manifest["metadata"]["project_name"], dbt_resource_props["original_file_path"]
         )
         node_ast = parse_one(sql=node_sql_path.read_text()).expression
-        optimized_node_ast = cast(
-            exp.Query,
-            optimize(
-                node_ast,
-                schema=sqlglot_mapping_schema,
-                validate_qualify_columns=False,  # Don't throw an error if we can't qualify a column without ambiguity.
```

Comment:
Why take out `validate_qualify_columns`?

Reply:
It's on by default